bundles/org.openhab.binding.bluetooth/src/main/java/org/openhab/binding/bluetooth/BluetoothDevice.java

```diff
@@ -249,6 +250,26 @@ public BluetoothAdapter getAdapter() {
      */
     public abstract boolean writeCharacteristic(BluetoothCharacteristic characteristic);
 
+    /**
+     * Returns if notifications can be enabled on the given characteristics.
+     *
+     * @param characteristic the {@link BluetoothCharacteristic} to check if notifications can be enabled.
+     * @return true if notifications can be enabled, false if notifications are not supported, characteristic is missing
+     *         on device or notifications are not supported.
+     */
+    public boolean canNotify(BluetoothCharacteristic characteristic) {
+        return characteristic.hasPropertyEnabled(BluetoothCharacteristic.PROPERTY_NOTIFY);
+    }
```

Comment:
Instead of having this method on the device I'd prefer if you just added a convenience method to `BluetoothCharacteristic` instead.